models/classes/runner/time/Timer.php

```diff
@@ -22,6 +22,8 @@
 
 namespace oat\taoTests\models\runner\time;
 
+use oat\taoQtiTest\models\runner\time\TimerStrategyInterface;
```

Comment:
it's impossible to use taoQtiTest in the taoTest, you need to move this interface to taoTest

Reply:
TimerStrategyInterface moved to extension-tao-test